ee/i18n/en.i18n.json

```
@@ -0,0 +1,61 @@
+{
+  "error-canned-response-not-found": "Canned Response Not Found",
+  "error-max-number-simultaneous-chats-reached": "The maximum number of simultaneous chats per agent has been reached.",
+  "Add_monitor": "Add monitor",
+  "Available_departments": "Available Departments",
+  "Canned Responses": "Canned Responses",
+  "Canned_Response_Removed": "Canned Response Removed",
+  "Canned_Responses_Enable": "Enable Canned Responses",
+  "Edit_Tag": "Edit Tag",
+  "Edit_Unit": "Edit Unit",
+  "Enterprise_License": "Enterprise License",
+  "Failed_to_add_monitor": "Failed to add monitor",
```

```suggestion
  "Failed_to_add_monitor": "The selected user is not a monitor",
```
I think we can improve it, it seems to be too technical to final users.
Maybe we can do something like we were doing on the old `Enterprise` version:
https://github.com/RocketChat/Rocket.Chat.Enterprise/blob/542717863ef8dd947f147d604d468b3befafb2ce/packages/rocketchat-enterprise-i18n/i18n/en.i18n.json#L36

---

they're not the same.. "Failed_to_add_monitor" is being used when an unknown error happens.. so I think it's fine